content/applications/sales/members.rst

```diff
@@ -0,0 +1,91 @@
+=======
+Members
+=======
+
+The *Members* application is where all operations related to memberships can be configured and
```

Comment:
```suggestion
The **Members** application is where all operations related to memberships can be configured and
```
using **BOLD** for app names now, based on most recent RST guideline directives

Reply:
Please implement this suggestion, as it is now a requirement when it comes to formatting application names. Thanks